src/app/aside/aside.component.scss

```diff
@@ -10,8 +10,8 @@
 
 .nav li a {
 
-  padding-left: 24px;
-  margin-bottom: 1px;
+  padding-left: 1rem;
```

Comment:
@Yuriy-27 I'm not sure but I think 1 rem in your app is 16 px by default.
So you can use some not integer values if needed like `.75rem` or even `0.375rem`

just FYI.